zerver/openapi/zulip.yaml

```diff
@@ -11308,6 +11308,27 @@ paths:
                           **Changes**: New in Zulip 7.0 (feature level 164). Clients should use 140
                           for older Zulip servers, since that's the value that was hardcoded in the
                           Zulip client apps prior to this parameter being introduced.
+                      server_typing_started_expiry_period:
+                        type: integer
+                        description: |
+                          How long before we assume a client has gone away
+                          and expire its typing status.
```

Comment:
Let's have these descriptions link to https://zulip.com/api/set-typing-status to explain how they're used. That page's `zulip.yaml` code also needs an update to reference these.

Also need to provide the units of milliseconds in this description.

And I think each of these "Changes" entries should state the previously hardcoded values. E.g. "New in Zulip 8.0 (feature level 202). Clients should use the previously hardcoded value of 10000 for older servers."

Reply:
Actually let's just rename it to `server_typing_started_expiry_period_milliseconds`  both here and in the settings while you're it.